website/src/pages/summarize/story.tsx

```diff
@@ -0,0 +1,107 @@
+import { Textarea } from "@chakra-ui/react";
```

Comment:
Can you add a comment at the top here:

`// TODO(#65): Unify and simplify the task paths.`